labs/03_selenium/signup.py

```diff
@@ -0,0 +1,54 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
+import time
+
+APP_URL = "http://localhost:5173"
+
+options = Options()
+options.add_experimental_option("excludeSwitches", ["enable-logging"])
+
+
+def test_sign_up():
+    # Arrange
+    driver = webdriver.Chrome(options=options)
+    driver.get(APP_URL)
+
+    sign_up_btn = driver.find_element(By.XPATH, "//button[text()='Sign Up']")
+    sign_up_btn.click()
+
+    username_input = driver.find_element(By.XPATH, "//input[@placeholder='Username']")
+    username_input.send_keys("Hola")
```

Comment:
This test would run one time only, because on the second execution the user Hola would exist already in the DB.

You could create a function to generate a random string, same as we did in Postman.